config/default/core.entity_form_display.taxonomy_term.h_notification_type.default.yml

```diff
@@ -27,6 +29,13 @@ content:
     third_party_settings: {  }
     type: options_select
     region: content
+  field_h_icon:
+    type: media_library_widget
```

Comment:
The icon field should use an entity browser widget for consistency with other media/file selection fields. You will most likely have to create a new entity browser instance specifically for use of SVG images.